package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
I just tried this change, but it causes a [test failure in Node 4.7.2](https://travis-ci.org/Polymer/polymer-build/jobs/191794387#L191) (at least for the `polymer-build` repo):

```shell
[23:08:04] SyntaxError in plugin 'gulp-mocha'
Message:
    Unexpected token ...
Stack:
SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at /home/travis/build/Polymer/polymer-build/node_modules/gulp-mocha/node_modules/mocha/lib/mocha.js:220:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/travis/build/Polymer/polymer-build/node_modules/gulp-mocha/node_modules/mocha/lib/mocha.js:217:14)
```